endpoints/oauth/test/test_login.py

```diff
@@ -113,7 +126,13 @@ def test_flagged_user_creation(
         with patch("features.INVITE_ONLY_USER_CREATION", invite_only):
             # Conduct login.
             result = _conduct_oauth_login(
-                internal_auth, login_service, login_service_lid, login_service_lid, email
+                app,
+                analytics,
```

Comment:
Do we need to pass this analytics objects around, are we actually using it?

Reply:
yes, the analytics object is used to count successful creations of a federated login. Not sure where we actually hook it into though ... https://github.com/quay/quay/blob/5defae13285881e582b2fa69eab3084d853bc38e/oauth/login_utils.py#L224